app/views/layouts/listnav/_host.html.haml

```diff
@@ -9,7 +9,7 @@
         - height = 90
       - else
         - height = 78
-      = render :partial => "layouts/quadicon", :locals => {:mode => :icon, :item => @record, :size => 72, :typ => :listnav, :height => height}
+      = render_quadicon(@record, :mode => :icon, :size => height, :typ => :listnav)
```

Comment:
This seems wrong.. with `:size => 72` and `:height => 78` (or 90), you'd get style `height: 78px`, pass `:size => 72`  and use `72/*.png`, with this, you'll get `height: 80px`, pass `:size => 78` and use `78/*.png` (and there's no `app/assets/images/78/`)